validphys2/src/validphys/core.py

```diff
@@ -472,7 +436,8 @@ def __init__(self, *, name, commondata, fkspecs, thspec, cuts,
 
     @functools.lru_cache()
     def load(self):
-        cd = self.commondata.load()
+        """Load the libNNPDF version of the dataset"""
+        cd = LegacyCommonData.ReadFile(str(self.commondata.datafile), str(self.commondata.sysfile))
```

Comment:
Yes. I think the last dependency will be dropped by #1651, but now after applying the changes I'll check whether a `load` is left anywhere else. Maybe it can be removed also from here.